src/Fixer/Phpdoc/PhpdocOrderFixer.php

```diff
@@ -118,8 +118,12 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
         $configurationOrder = [];
         foreach ($this->configuration['order'] as $type) {
             $configurationOrder[] = $type;
-            $configurationOrder[] = 'phpstan-'.$type;
-            $configurationOrder[] = 'psalm-'.$type;
+            if (!\in_array('phpstan-'.$type, $this->configuration['order'], true)) {
+                $configurationOrder[] = 'phpstan-'.$type;
+            }
+            if (!\in_array('psalm-'.$type, $this->configuration['order'], true)) {
+                $configurationOrder[] = 'psalm-'.$type;
+            }
```

Comment:
@mvorisek are you happy with state of this PR?

Reply:
yes